lib/scrub/record_counter.rb

```diff
@@ -1,18 +1,24 @@
+require "scrub/member_holding_file"
```

Comment:
This `require` not needed? Unless `count_distinct_ocns` is intended to move there?

Reply:
Indeed, this is a remnant from an abandoned idea. Removed.